Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -259,13 +260,35 @@ void MaxEnt::init() {
                   "The output workspace containing the reconstructed data.");
 }
 
+//----------------------------------------------------------------------------------------------
+/** Revisit bin edge validation to write errors/warnings to the log
+ */
+void MaxEnt::validateBinEdges(std::string wsName,
+                              std::map<std::string, std::string> &messages) {
+  const double warningLevel = 0.01;
+  MatrixWorkspace_sptr ws = getProperty(wsName);
```

Comment:
Can this be `auto const`?

Reply:
It can't be auto because getProperty returns Mantid::Kernel::IPropertyManager::TypedValue which doesn't support the use as a boolean value on the next line. The use of the explicit type is casting it to a shared_ptr. Will change to const though